renderer/index.css

```diff
@@ -827,6 +827,11 @@ body.drag .app::after {
   outline: none;
 }
 
+.player-controls .time {
```

Comment:
This currently causes the play button to be off centre if the length of the progress time text is long and the window size is small. Not too sure how to circumvent that though.